backend/app/controllers/api/v1/open_calls_controller.rb

```diff
@@ -0,0 +1,29 @@
+module API
+  module V1
+    class OpenCallsController < BaseController
+      include API::Pagination
+
+      def index
+        open_calls = OpenCall.all.includes(:investor)
+        pagy_object, open_calls = pagy(open_calls, page: current_page, items: per_page)
+        render json: OpenCallSerializer.new(
+          open_calls,
+          links: pagination_links(:api_v1_open_calls_path, pagy_object),
+          meta: pagination_meta(pagy_object)
+        ).serializable_hash
+      end
+
+      def show
+        open_call = fetch_open_call
```

Comment:
I would prefer `before_action` maybe here

Reply:
If there are more actions then yes.

Reply:
sure, no problem. From my experience use case is often that you start with API controller which has 1 or 2 actions and it often grows later on, so it is better to take into account that it will grow later on then in middle to switch to `before_action` and refactor already existing code ;)

Reply:
Well, yeah, I know we will have create/update actions as well, so I can move it. I just was too lazy to do it now.